src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -349,6 +350,93 @@ export const exchangeStitchingEnvironment = ({
           },
         },
       },
+      Mutation: {
+        // monkey-patch the existing exchange mutation
```

Comment:
instead of monkey-patching, this mutation can be excluded from being included with the Exchange schema (via [this transform](https://github.com/artsy/metaphysics/blob/897f4b5985282d3e6b99de87f6a7b1b586f46707/src/lib/stitching/gravity/schema.ts#L52), and then this mutation can be defined more conventionally inside MP.

Reply:
We started out with that approach, then decided it might be more clear to follow the exchange convention. So something like `Mutation.createInquiryOffer` but defined in this stitching file?

Reply:
I'm not following the stitching use-case exactly. Per https://github.com/artsy/metaphysics/pull/2974/files#r570581737 basically, I would probably stick with a regular Metaphysics mutation at this point.